test/unit/processing/delta/calculate-delta.test.js

```diff
@@ -259,7 +260,7 @@ describe('calculate delta', () => {
     const previousPaymentRequests = [{
       ledger: AP,
       value: 200,
-      settled: new Date(2022, 3, 5),
+      settledValue: 0,
```

Comment:
Should this not be settledValue: 200 to correspond with test above?

Reply:
It technically doesn't change the scope of the test as we're not testing ledger assignment but I agree it reduces the variables if we assume fully settled and we're consistent with the equivalent test as you say.  Updated.